src/main/java/com/ncc/aif/ont2javagen/OntologyGeneration.java

```diff
@@ -33,11 +33,7 @@ private OntologyGeneration() {
         this.classesToGenerate = new ArrayList();
```

Comment:
What is classesToGenerate used for?  It's declared, initialized, and URIs are added to it, but then it's never used that I can tell.

Reply:
Code was vestigial and has been removed.